src/logics/ArrayTheory.cc

```diff
@@ -5,43 +5,11 @@
  */
 
 #include "ArrayTheory.h"
+
```

Comment:
I think this space is not needed here

Reply:
This follows the Google guidelines: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

It visually separates the header corresponding to this .cc file from other includes.